arm_robots/src/arm_robots/config/med_config.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
```

Comment:
since you're not starting from existing code like we were, I'd encourage you to get rid of this file if possible. Most of this stuff is already available on the parameter server, so looking it up would make things simpler and reduce duplication of hard-coded values. Look at the yaml/config files, or srdf file for where this stuff might belong

Reply:
Do you guys have a plan to shift away from this setup? I can imagine adding a `get_joint_limits()` function to the `MoveItEnabledRobot` class to do this.

Reply:
Then each subclass would need to implement it according to their arms.

Reply:
ARM_JOINT_NAMES is the most problematic part. That should just be based on a move_group., which you can get via the robot_commander. But that breaks the assumption that BaseVictor (for instance) doesn't depend on MoveIt. So calling it MoveitEnabledRobot doesn't make sense, not sure what the distinction between Base and Movet robot should be, if indeed there should be one at all?

as for the vel/accel limits, again those can be retreived from the parameter server, and there's probably API on RobotCommander or somewhere else in moveit to get that info

Ideally our `get_joint_limits` would be a wrapper to some existing moveit function call? not sure

Reply:
Ok, well for vel/accel limits, it seems to make sense to have that in MoveItEnabledRobot and use the RobotCommander or similar to get access. Maybe that can be a separate PR?

It's not immediately clear to me how to address the ARM_JOINT_NAMES issue, but it also seems like something both Victor/Med suffer from. Maybe enough for now to open an issue and explore options later?

Reply:
Already is an issue:
https://github.com/UM-ARM-Lab/arm_robots/issues/14

I agree this change is tangential to your refactor, and should be solved in a separate PR

Reply:
agreed